viewing-data.html

```diff
@@ -432,15 +432,31 @@
 
 <p>Expand the <span class="bold">Screen</span> folder, expand the <span class="bold">Plate</span> container and select a <span class="bold">Run</span>.</p>
 
-<p>Click on a well in the plate shown on the centre pane to view the data in that well.</p>
+<p>Click on a well in the plate shown in the centre pane to view the data in the well. Well annotations, but no image annotations, are displayed in right-hand pane. Preview for wells is not available.</p>
```

Comment:
"but *not* image ann.." might be better.
Also "Preview for Wells is not *currently* available" would suggest that we are working on improving this.